src/dev-dashboard/ui/src/slate-grid.css

```
@@ -262,6 +262,86 @@
     padding: 0.2rem 0.9rem;
 }
 
+/* ── Obsidian callouts (rendered by obsidianCalloutExtension) ─────────── */
+.dd-markdown .markdown-alert {
+    --cal: #3c4a44;
+    margin: 1rem 0;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)